ansible/playbook/tasks/update.yml

```diff
@@ -308,12 +316,13 @@
           - pmm-agent
         - ["stop", "remove", "add"]
 
-    - name: Delete old folder for grafana plugins
+    - name: Delete old grafana plugin folders and old grafana database
```

Comment:
IMO it's a wrong place for such an action since it originates from the [spec](https://github.com/percona/pmm-server/blob/main/rhel/SPECS/grafana.spec), so the spec would be the right place to remove the db (or not to copy it at all).